packages/components/src/LightBox/LightBox.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { LightBox } from ".";
+
+afterEach(cleanup);
+
+test("Lightbox opens and shows the image", () => {
+  const title = "Dis be a title";
+  const caption = "Dis be a caption 🎉";
+  const handleClose = jest.fn();
+
+  const { queryByText } = render(
+    <LightBox
+      open={true}
+      images={[
+        {
+          title: title,
+          caption: caption,
+          url: "https://i.imgur.com/6Jcfgnp.jpg",
+        },
+      ]}
+      onRequestClose={handleClose}
+    />,
+  );
+
+  expect(queryByText(title)).toBeTruthy();
```

Comment:
```suggestion
  expect(queryByText(title)).toBeInstanceOf(HTMLElement);
```

Reply:
Might not be a bad idea too to do a quick test that the `handleClose` gets called properly as well.

Reply:
Yeah this makes more sense. This is me still being a newb in this test library. I originally had `expect(queryByText(title)).toHaveText(title);` but that felt redundant. I then went through `modal.test.tsx` and this is how we do this test there FYI.